### PR TITLE
styled the bookstore

### PR DESCRIPTION
**Pull request by Samuelwanza**
I have conducted the following:
-Styled the components of the bookstore using modular css
-Ensured the functionality of the bookstore is maintained

### CODE REVIEW

File: src/styles/Layout.module.css [hunk 1]
```diff
@@ -1,30 +1,60 @@
...
+  padding-right: 50px;
+  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.1);
 }
 
 .container {
+  width: 100%;
+  background: #f5f6fa;
+}
+
+.navigation {
```
Comment (andersonlebon) [inline src/styles/Layout.module.css:19]: - [ ] Kindlynote that the styling  should be  **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). you need to use the same layout, font-size dimentions, positions of items, colors, ,margins, paddings...

> Your design 
>  image
>The template
> image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Samuelwanza 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- Nice use of style module :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/Layout.module.css (hunk 1 above)
Comment (Lucas-Masaba) [reply src/styles/Layout.module.css:19]: - [ ] Like the previous reviewer mentioned, please make sure the design is the same as that seen in the original design on Zeplin. 
  * Please add the line above the "ADD NEW BOOK" form.
  *  Please make sure the "ADD BOOK"  button is aligned with the cards containing the books' information.
  * The input fields should be closer to each other as seen in the original design
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Samuelwanza 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost-there

 ## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made most changes requested by the previous reviewer :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Samuelwanza 

Your project is complete! Great job  🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**